tests/framework/installer/installer.go

```diff
@@ -46,6 +46,8 @@ var (
 	baseTestDir       string
 	forceUseDevices   = false
 	createBaseTestDir = true
+	// RookTestOpenshift variable which define testing on openshift
+	RookTestOpenshift = os.Getenv("ROOK_TEST_OPENSHIFT") == `true`
```

Comment:
Instead of adding RookTestOpenshift var here, how about adding it to the EnvironmentManifest struct [here](https://github.com/rook/rook/blob/master/tests/framework/installer/environment.go)? Then we can unify the new env vars with the existing ones.